google-cloud-language/lib/google/cloud/language/v1/credentials.rb

```diff
@@ -21,6 +21,7 @@ module Language
       module V1
         class Credentials < Google::Auth::Credentials
           SCOPE = [
+            "https://www.googleapis.com/auth/cloud-language",
```

Comment:
I dislike this. The way this is implemented users need both of these scopes. But the intention is that a user could use either. Requiring both by default seems excessive.

Reply:
I don't disagree, but it's a common pattern, both here (talent, vision, etc) and on other languages.

Reply:
This is something we'll eventually need to fix, across the libraries. Given that, I'm not too worried about it changing here for now.